app/mailers/notification_mailer.rb

```diff
@@ -32,4 +32,10 @@ def submitted_for_review_email
     @work = params[:work]
     mail(to: @user.email, subject: "A Depositor has submitted a deposit in the #{@work.collection_name} collection")
   end
+
+  def invitation_to_deposit_email
```

Comment:
Can you move this out of the NotificationMailer and move it to a "CollectionsMailer"?  Currently the notifications mailer is all about events on a work, and this new message is about events on a collection.  It seems like this would make it easier to find later.

Reply:
yeah, i can do that -- though then we want to rename `NotificationMailer` to `WorksMailer` for consistency.

Reply:
That sounds like a splendid idea.

Reply:
Done in #747